eng/pipelines/templates/steps/analyze.yml

```diff
@@ -110,6 +110,14 @@ steps:
     failOnStderr: true
     workingDirectory: 'sdk/${{parameters.ServiceDirectory}}'
 
+  - task: PowerShell@2
+    displayName: 'Ignore Mgmt README.md Link'
```

Comment:
Can you change the displayName to `Add module link in README.md to 'ignore-links.txt'`?